apps-rendering/src/item.ts

```diff
@@ -126,7 +126,33 @@ interface Correction extends Fields {
 	design: ArticleDesign.Correction;
 	body: Body;
 }
+interface Interview extends Fields {
+	design: ArticleDesign.Interview;
+	body: Body;
+}
+
+interface Quiz extends Fields {
+	design: ArticleDesign.Quiz;
+	body: Body;
+}
+interface Recipe extends Fields {
+	design: ArticleDesign.Recipe;
+	body: Body;
+}
 
+interface Feature extends Fields {
+	design: ArticleDesign.Feature;
+	body: Body;
+}
+interface PhotoEssay extends Fields {
+	design: ArticleDesign.PhotoEssay;
+	body: Body;
+}
+
+interface PrintShop extends Fields {
+	design: ArticleDesign.PrintShop;
+	body: Body;
+}
```

Comment:
We may want to a) add these to the `Exclude` statement on `Standard` so that we're not specifying them in two places, or b) consider committing to getting rid of the `Exclude` statement entirely and having an `interface` for each `Design`.

This is perhaps not directly related to storybook layouts so something for a follow-up. Perhaps we should chat about it?

Reply:
Yes, let me card it up!